packages/resource-deployment/README.md

```diff
@@ -28,20 +28,28 @@ The deployment script is going to create Resource Group and deploy Storage Accou
     az login
     ```
 
-### 4. Clean
+### 4. Build repository
+
+-   Run the below command to build the whole repo
```

Comment:
>Run the below command to build the whole repo [](start = 4, length = 45)

Can we update it to:
Run the below command from repository root folder

Reply:
not needed. I am using --cwd to do that

---
In reply to: [296421624](https://github.com/microsoft/accessibility-insights-service/pull/174#discussion_r296421624) [](ancestors = 296421624)

Reply:
fixed

---
In reply to: [296422374](https://github.com/microsoft/accessibility-insights-service/pull/174#discussion_r296422374) [](ancestors = 296422374,296421624)